packages/eslint-plugin/typings/ts-eslint.d.ts

```diff
@@ -262,7 +262,7 @@ declare module 'ts-eslint' {
     /**
      * The name of the rule this rule was replaced by, if it was deprecated.
      */
-    replacedBy?: string;
+    replacedBy?: string[];
```

Comment:
It seems that `replacedBy` has `string[]` type.
https://eslint.org/docs/developer-guide/working-with-rules#rule-basics